airsonic-main/src/main/resources/application.properties

```diff
@@ -28,3 +28,6 @@ spring.task.scheduling.pool.size=10
 # airsonic config
 airsonic.cue.enabled=true
 airsonic.cue.hide-indexed-files=true
+
+server.servlet.session.persistent= true
```

Comment:
Persisting the session allows restarting the server without forcing a new login. This is very useful for development, but also a good idea in production